DataFormats/SiPixelClusterSoA/src/alpaka/classes_serial_def.xml

```diff
@@ -0,0 +1,18 @@
+<lcgdict>
+<class name="PortableHostCollection<SiPixelClustersLayout>"/>
```

Comment:
Please move the host collection dictionary declarations from `alpaka/classes_serial{.h,_def.xml}` to `classes{.h,_def.xml}` (the placement convention was updated).